documentation/doxygen/Makefile

```diff
@@ -53,6 +53,7 @@ doxygen: filter pyzdoc
 	./makeinput.sh
 	doxygen
 	./listlibs.sh
+	sed -i -e '/^ */s///' -e '/ *$$/s///'  $(DOXYGEN_OUTPUT_DIRECTORY)/html/index.qhp
```

Comment:
qhelpgenerator $(DOXYGEN_OUTPUT_DIRECTORY)/html/index.qhp

Reply:
done.

Reply:
Looks good now! I do not know if we will have to 'mv' the output file then by hand to the doxygen output directory, or if it generates it directly there.
Probably we also need to rename it from index.qch to ROOT.qch, (which doxygen does behind the scenes after running qhelpgenerator)

Reply:
index.qch is already in the right place. No need to move it. 
Ah yes will rename it also.

Reply:
> index.qch to ROOT.qch

you mean `.qhp` I guess ?

Reply:
qhelpgenerator transforms a qhp input file into a qch output file. I guess either we rename the input file, or the output file, both strategies should work afaik.

Reply:
Oh ok ... does the `.qch` disappear ?

Reply:
Yes, the qhp is only an auxiliary file, so to say the intermediary file between doxygen and qt. qthelpgenerator transforms it to "binary" so to say. I think it does not disappear by itself, though we could remove it to save space if needed.

Reply:
Ok I rename the the `.qch` now